src/components/installation/InstallationProgress.jsx

```diff
@@ -45,27 +45,27 @@ import "./InstallationProgress.scss";
 
 const _ = cockpit.gettext;
 const N_ = cockpit.noop;
-const idPrefix = "installation-progress";
+const idPrefix = "progress";
```

Comment:
I'm thinking that perhaps if this is now essentially a part of the stable API (as people might have their own config files via the drop-in dir) maybe it should actually have some prefix ?

Eq. `anaconda-screen-foo` or `anaconda-feature-foo`

That would also make it easier to find all places where it is used & possibly make it less likely to be broken by mistake. 

And lastly if we need to change such a string in the future (eq. splitting a screen or changing its purpose) while keeping backwards compatibility, it would be easier with a well-known prefixed id than just a generic name.

In any case, this is the best time to think about it, while we can be reasonably sure no one is using this yet :)